cypress/features/regression/carousel.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Carousel component
+  I want to change Carousel properties
+
+  Background: Open Carousel component page
+    Given I open "Carousel" component page
+
+  @positive
+  Scenario: Initial slide title is Slide 3
+    # When I open "Carousel" component page
+    Then slide 2 title is "Slide 3"
+
+  @positive
+  Scenario Outline: Set slide index to <index>
+    When I select slideIndex to "<index>"
+    Then slide <index> title is "<title>"
+    Examples:
+      | index | title                |
+      | 0     | Slide 1              |
+      | 1     | Full clickable slide |
+      | 2     | Slide 3              |
+      | 3     | Slide 4              |
+      | 4     | Slide 5              |
+
+  # @positive
```

Comment:
we need to uncomment those tests or just delete them

Reply:
Sorry, I'd commented them out so only this one scenario outline was being run til I fixed the issue with the slide title selection to save time. The tests are still required :)